src/utils/evm-networks.ts

```diff
@@ -0,0 +1,102 @@
+import { message } from 'antd';
+import { Blockchain } from '@liqnft/candy-shop-types';
+
+export enum ChainId {
+  MATIC = 137,
+  MUMBAI = 80001,
+  ETH = 1,
+  GOERLI = 420,
```

Comment:
`5`
ref: https://besu.hyperledger.org/en/stable/public-networks/concepts/network-and-chain-id/